timemachine/fe/atom_mapping.py

```diff
@@ -343,19 +349,31 @@ def _get_cores_impl(
     n_a = len(conf_a)
     n_b = len(conf_b)
 
-    chiral_set_a = ChiralRestrIdxSet.from_mol(mol_a, conf_a)
-    chiral_set_b = ChiralRestrIdxSet.from_mol(mol_b, conf_b)
+    filter_fxns = []
+    if enforce_chiral:
+        chiral_set_a = ChiralRestrIdxSet.from_mol(mol_a, conf_a)
+        chiral_set_b = ChiralRestrIdxSet.from_mol(mol_b, conf_b)
 
-    def chiral_filter(trial_core):
-        # TODO: avoid conversion
-        np_core = np.array([(i, trial_core[i]) for i in range(len(trial_core)) if trial_core[i] != -1])
-        passed = not has_chiral_atom_flips(np_core, chiral_set_a, chiral_set_b)
-        return passed
+        def chiral_filter(trial_core):
+            # TODO: avoid conversion
+            np_core = np.array([(i, trial_core[i]) for i in range(len(trial_core)) if trial_core[i] != -1])
+            passed = not has_chiral_atom_flips(np_core, chiral_set_a, chiral_set_b)
+            return passed
 
-    if enforce_chiral:
-        filter_fxn = chiral_filter
-    else:
-        filter_fxn = lambda core: True
+        filter_fxns.append(chiral_filter)
+
+    if disallow_planar_torsion_flips:
+        find_flipped_planar_torsions = setup_find_flipped_planar_torsions(mol_a, mol_b)
+
+        def planar_torsion_flip_filter(trial_core):
+            flipped = find_flipped_planar_torsions(trial_core)
+            passed = flipped == []
```

Comment:
nit:

```suggestion
            passed = len(flipped) == 0
```

Reply:
Changed in https://github.com/proteneer/timemachine/pull/1229/commits/fe75bf3b84217bdace0d630320aa824b90cd711c